modules/cloud-create-instance-steps.adoc

```diff
@@ -20,9 +20,11 @@ To create an *ACS instance*:
 . From the navigation menu, select *Advanced Cluster Security* -> *ACS Instances*.
 . Select *Create ACS instance* and enter information into the displayed fields or select the appropriate option from the drop-down list:
 * *Name*: Enter the name of your *ACS instance*. An *ACS instance* contains the {product-title-short} Central component, also referred to as "Central," which includes the {product-title-managed-short} management interface and services that are configured and managed by Red Hat. You manage your secured clusters that communicate with Central. You can connect multiple secured clusters to one instance.
-* *Cloud provider*: The cloud provider where Central is located. Select *AWS*. 
-* *Cloud region*: The region for your cloud provider where Central is located. Select *AWS*.
-* *Availability zones*: Use the default value.
+* *Cloud provider*: The cloud provider where Central is located. Select *AWS*.
+* *Cloud region*: The region for your cloud provider where Central is located. Select one of the following:
+** US-East, N. Virginia
+** Europe, Frankfurt
```

Comment:
This has been updated to Dublin already (eu-west-1), the new text is "Europe, Ireland"

![image](https://user-images.githubusercontent.com/497744/228855240-e993da82-e8cf-4d2b-8e9d-4278bc95a14b.png)